### PR TITLE
Code review cleanup

### PR DESCRIPTION
**Pull request by ganadurai**
## Description
What's changed, or what was fixed?

Addressed the fixes for https://github.com/apigee/devrel/issues/599 , except point #3, waiting for further comments on it

## Housekeeping
(please check all that apply [x], do not edit the text)
- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

## Full Repo Validation Required
(please check all that apply [x], do not edit the text)
- [ ] PR requires full pipeline run (Run for changes only by default).

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

File: tools/apigee-envoy-quickstart/README.md [hunk 1]
```diff
@@ -50,20 +52,25 @@ The Apigee Envoy Quickstart Toolkit sets up the Envoy proxies with Apigee adapte
...
-    wget -O devrel.zip https://github.com/apigee/devrel/archive/refs/heads/main.zip
-    unzip devrel.zip
-    mv devrel-main apigee-devrel
-    rm devrel.zip
-    cd ${ENVOY_HOME}/apigee-devrel/tools/apigee-envoy-quickstart/
+    ```
+
+1. **Clone Apigee DevRel repository, if not exists.** 
+    ```bash
+    git clone https://github.com/apigee/devrel/archive/refs/heads/main.zip
```
Comment (danistrebel) [inline tools/apigee-envoy-quickstart/README.md:63]: This doesn't work

**Review by danistrebel — CHANGES_REQUESTED** [review]

File: tools/apigee-envoy-quickstart/README.md (hunk 1 above)
Comment (ganadurai) [reply tools/apigee-envoy-quickstart/README.md:63]: duh.... stupid mistake. .. will fix it.

**Review by danistrebel — APPROVED** [review]
Thanks @ganadurai

**Pull request merged**